plugins/rplanners/ParabolicPathSmooth/ParabolicRamp.cpp

```diff
@@ -2898,7 +2902,7 @@ Real SolveMinTimeBounded(const Vector& x0,const Vector& v0,const Vector& x1,cons
                     solved = false;
                     if( 1 ) { // disable to avoid changing time
                         PARABOLIC_RAMP_PLOG("ParabolicRamp1D info: x0 = %.15e; x1 = %.15e; v0 = %.15e; v1 = %.15e; vm = %.15e; am = %.15e; endTime = %.15e", x0[i], x1[i], v0[i], v1[i], vmax[i], amax[i], endTime);
-                        Real newEndTime;
+                        Real newEndTime = 0.0;
```

Comment:
@Puttichai could you please take a look, what would be the proper default value? there are code paths (ramps.size() == 0) that skip assigning the value, but it's used later in calculations.

Reply:
The entire section is inside the outer for-loop `for(size_t i=0; i<ramps.size(); i++)`, so I think this is fine too.